components/src/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,282 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { Manager, Reference, Popper } from "react-popper";
+import theme from "../theme";
+/* eslint react/destructuring-assignment: 0 */
```

Comment:
2 questions here:
- Why is there a special rule for this file?
  - We should set eslint rules in this order of preference:
    - Apply globally
    - Disable for specific lines of code using disable/enable
      - https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments
    - Disable for specific file
- Why isn't it closer to where it affects?